agent/test-agent-cli/design/RUNBOOK.md

```diff
@@ -0,0 +1,99 @@
+# Steps to use Test-agent Command Line Interface
```

Comment:
Yesterday we merged a new `make mdlint` target that performs markdown linting. There are a bunch of things here that the linting is going to flag. Can you rebase on the latest from `develop` and run the mdlint target to help clean up things. They are all very minor formatting things.

Reply:
As I had discussion with @stmcginnis, the cleanup will be done in the [#514](https://github.com/bottlerocket-os/bottlerocket-test-system/pull/514)